docs/configuration/features/text_features.md

```diff
@@ -384,17 +386,7 @@ loss:
     class_weights: 1
     class_similarities_temperature: 0
 decoder: 
-    type: tagger
-    num_fc_layers: 0
-    output_size: 256
-    use_bias: true
-    weights_initializer: glorot_uniform
-    bias_initializer: zeros
-    activation: relu
-    dropout: 0
-    attention: false
-    attention_embedding_size: 256
-    attention_num_heads: 8
```

Comment:
Should we add these back since this block represents an example with all the defaults?

Reply:
It's in the decoder section below.